QUARKUS-4550.md

```diff
@@ -0,0 +1,68 @@
+# QUARKUS-4550 - Enable proxy configuration for OpenTelemetry exporters
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-4550
+
+Upstream issues: https://github.com/quarkusio/quarkus/issues/39519, https://github.com/quarkusio/quarkus/issues/39729
+
+Upstream PRs: https://github.com/quarkusio/quarkus/pull/39543, https://github.com/quarkusio/quarkus/pull/39733
+
+Quarks OpenTelemetry extension has builtin Vert.x-based OTLP exporter for gRPC and `http/protobuf` protocols.
+Underneath Vert.x HTTP client is used that supports proxy options.
+With this change, proxy options are now propagated when Quarkus-specific properties are set.
+Additionally, when JDK's own HTTP / HTTPS proxy system properties are set, Quarkus respects them as well.
+
+## Scope of the testing
+Let's consider all testing options:
```

Comment:
LGTM, but after reading all Scope of Testing, I'd rather describe what will be tested and what won't be tested and why instead of your description approach, but it's up to you to change it.

Reply:
well, yeah, it is rather unorthodox approach. I have never done it before. I don't mind rewriting it, I just wanted to be sure that reviewers understand what I am going to NOT DO (in case they thought there is nothing else do do).

let me try again.

Reply:
@jcarranzan I rewrote this part, does it look better? We can iterate as long as you wish. Let's comment on updated text so I'll close this comment thread